flexx/app/_server.py

```diff
@@ -33,9 +34,24 @@ def create_server(host=None, port=None, new_loop=False, backend='tornado'):
             which is made current when ``start()`` is called. If ``False``
             (default) will use the current IOLoop for this thread.
         backend (str): Stub argument; only Tornado is currently supported.
+        **server_kwargs: keyword arguments passed to the server constructor.
     
     Returns:
         server: The server object, see ``current_server()``.
+
+    Examples:
+
+        Configuring the server to use HTTPS with Tornado server:
+    
+        .. code-block:: py
+        
+            app.create_server(ssl_options = {'certfile' : '/path/to/certfile',
+                                             'keyfile' : '/path/to/keyfile'})
+            app.serve(Example, 'Example')
+            app.run()
+
+        Alternately, cert and key files can be provided through
+           ``ssl_certfile`` and ``ssl_keyfile`` configuration variables.
```

Comment:
I like the idea of setting ssl options via the config. In fact, I think we should advocate that as *the* way to do it. We can keep the `server_kwargs`, but can you please please remove the example in this docstring (I know I asked you to add it earlier, sorry!).